src/v2/view-builder/internals/BaseForm.js

```diff
@@ -76,6 +76,36 @@ export default Form.extend({
      */
     const infoContainer= '<div class=\'o-form-info-container\'></div>';
     this.$el.find('.o-form-error-container').before(infoContainer);
+
+    this.addIdentifier();
+  },
+
+  /**
+   * Render user identifier below title, or if no title, render below message.
+   */
+  addIdentifier() {
+    const { identifier } = this.options.appState.get('user') || {};
+    if (!identifier) {
+      return;
+    }
+    if (!this.settings.get('features.showIdentifier')) {
+      return;
+    }
+
+    const header = this.$el.find('[data-se="o-form-head"]');
+    const isTerminalView = this.options.currentViewState.name === 'terminal';
```

Comment:
Let's use the constants defined in RemediationConstants instead of hardcoding 'terminal'